Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
+
+[dev-packages]
+pytest = "*"
+coverage = "*"
+pytest-cov = "*"
+ipython = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
Can we force it on something newer? Or not yet?

Reply:
I don't think we have a base image newer than 3.8. 

I want to entirely rethink how we handle the base image but that's a way down the road.